tests/integration/kubernetes/runtimeclass_workloads/confidential/unencrypted/Dockerfile

```
@@ -16,9 +16,9 @@ RUN apk add --no-cache curl openssh-server
 RUN /bin/sh -c \
     'ARCH=$(uname -m) && \
     [[ "${ARCH}" == "x86_64" ]] && \
-    curl -LO https://github.com/klauspost/cpuid/releases/download/v2.2.5/cpuid-Linux_x86_64_2.2.5.tar.gz && \
-    tar -xvzf cpuid-Linux_x86_64_2.2.5.tar.gz  -C /usr/bin && \
-    rm -rf cpuid-Linux_x86_64_2.2.5.tar.gz && \
+    curl -LO https://github.com/klauspost/cpuid/releases/download/v2.2.7/cpuid-Linux_x86_64_2.2.7.tar.gz && \
+    tar -xvzf cpuid-Linux_x86_64_2.2.7.tar.gz  -C /usr/bin && \
+    rm -rf cpuid-Linux_x86_64_2.2.7.tar.gz && \
```

Hi @fidencio what's about using a Variable for  `2.2.7` or `2.2.5` to reduce changes and only update the VERSION when necessary ?
```suggestion
local VERSION=2.2.7
curl -LO https://github.com/klauspost/cpuid/releases/download/v${VERSION}/cpuid-Linux_${ARCH}_${VERSION}.tar.gz && \
tar -xvzf cpuid-Linux_${ARCH}_${VERSION}.tar.gz  -C /usr/bin && \
rm -rf cpuid-Linux_${ARCH}_${VERSION}.tar.gz && \
```